archway/assetlist.json

```diff
@@ -203,6 +201,114 @@
         }
       ],
       "coingecko_id": "astrovault"
+    },
+    {
+      "description": "Wrapped ARCH",
+      "type_asset": "cw20",
+      "address": "archway1msc3fa4e2mh73y760qmjd3wfkzr98nl26f36xgt2xnhuw6ay3scs42zeue",
+      "denom_units": [
+        {
+          "denom": "cw20:archway1msc3fa4e2mh73y760qmjd3wfkzr98nl26f36xgt2xnhuw6ay3scs42zeue",
+          "exponent": 0
+        },
+        {
+          "denom": "wARCH",
+          "exponent": 18
+        }
+      ],
+      "base": "cw20:archway1msc3fa4e2mh73y760qmjd3wfkzr98nl26f36xgt2xnhuw6ay3scs42zeue",
+      "name": "Wrapped ARCH",
+      "display": "wARCH",
+      "symbol": "wARCH",
+      "traces": [
+        {
+          "type": "wrapped",
+          "counterparty": {
+            "chain_name": "archway",
+            "base_denom": "aarch"
+          },
+          "provider": "Archway"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/wArch.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/wArch.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/wArch.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/wArch.svg",
+          "theme": {
+            "primary_color_hex": "#000000"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Circle's USDC Noble on Archway ",
+      "denom_units": [
+        {
+          "denom": "ibc/43897B9739BD63E3A08A88191999C632E052724AB96BD4C74AE31375C991F48D",
+          "exponent": 0,
+          "aliases": ["uusdc"]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/43897B9739BD63E3A08A88191999C632E052724AB96BD4C74AE31375C991F48D",
+      "name": "USDC (Noble)",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "synthetic",
+          "counterparty": {
+            "chain_name": "forex",
+            "base_denom": "USD"
+          },
+          "provider": "Circle"
+        },
+        {
+          "type": "bridge",
```

Comment:
again, this bridge transfer is inaccurate. I recommend sticking with only the latest trace hop